internal/aws/cwlogs/pusher.go

```diff
@@ -146,14 +146,14 @@ func (batch *logEventBatch) isActive(targetTimestampMs *int64) bool {
 	return true
 }
 
-func (batch *logEventBatch) append(event *logEvent) {
-	batch.putLogEventsInput.LogEvents = append(batch.putLogEventsInput.LogEvents, event.inputLogEvent)
+func (batch *logEventBatch) append(event *LogEvent) {
```

Comment:
Perhaps less significant, but maybe `eventBatch` here?  Basically, strip `log` from everywhere that it is repetitive or unnecessary since the whole purpose of this package is to handle logs.